docs/2.5/admin-guide.md

```diff
@@ -82,6 +82,23 @@ ExecStart=/usr/local/bin/teleport start --config=/etc/teleport.yaml
 WantedBy=multi-user.target
 ```
 
+### Graceful Restarts
+
+If using the systemd service unit file above, executing `systemctl restart teleport` 
```

Comment:
For this to be true, systemd unit has to look like this:



https://github.com/gravitational/teleport/blob/master/examples/aws/terraform/auth-user-data.tpl#L102

So we have to update the example in the docs too:
https://github.com/gravitational/teleport/blob/master/examples/systemd/teleport.service

and refer to it here as well